lib/ansible/cli/galaxy.py

```diff
@@ -68,8 +68,16 @@
     ('v3', False),
     ('validate_certs', False),
     ('client_id', False),
+    ('timeout', False),
 ]
 
+SERVER_ADDITIONAL = {
+    'v3': {'type': 'bool', 'default': 'False'},
+    'validate_certs': {'type': 'bool', 'default': 'True', 'cli': [{'name': 'ignore_certs'}]},
+    'timeout': {'type': 'int', 'default': '60', 'cli': [{'name': 'timeout'}]},
+    'token': {'default': NoTokenSentinel},
```

Comment:
This fails in CI with a yaml.representer.RepresenterError because we use  safe_dump
```suggestion
    'token': {'default': None},
```